docs/tutorials/03-publishing-pilets.md

```diff
@@ -19,52 +19,121 @@ We designed Piral to be as simple as possible, while remaining as flexible and p
 
-The service can be found at https://feed.piral.io. Registration is possible by using a Microsoft Account to log into the service. We do not need any personal data besides the email / account id, which is provided after your approval.
+The service can be found at <https://feed.piral.io>. Registration is possible by using a Microsoft Account to log into the service. We do not need any personal data besides the email / account id, which is provided after your approval.
 
-Logging into the service you'll see the overview page. At this point nothing will be displayed as you have not created any feed yet.
+For our tutorials, we will utilize this publicly available community edition of the feed service.
 
 [^1]: For development purposes and usage. The exact limitations are subject to change. Please see details on our feed service website.
 
-## Creating a Feed
+## Overview
+In the Getting Started tutorial, we covered already the creation of a Piral instance and its first pilet. This tutorial will guide us through the steps how to publish our first pilet to the feed service and fetch it subsequently into the Piral instance.
 
-For publishing (or retrieving) pilets you will need a feed. A feed is like a container or folder. The link for creating the feed service can be found in the top menu.
+This quick start will show us how to
 
-![Feed Service Header](../diagrams/feed-service.png)
+1. Creating a **Feed** in the feed service
+2. Create an **API Key** required for publishing the pilet
+2. **Publish a pilet** to the feed service
+3. **Configure the Piral Instance** to fetch pilets from the feed service
 
-To create the feed we need to give it a proper (unique) name. The system will check if the provided name is still available. The description can only be seen by you - it is optional. The allowed hosts you can also leave blank for now (resolving to `*`).
+## Prerequisites
 
-When you are done just press "create" and wait for the operation to complete. You will now be feed details page, which shows you the available pilets.
+For completing the tutorials, the following prerequisites must be met:
 
-## Creating an API Key
+- **Pilet** - we will use the sample pilet, which we have created in the Getting Started tutorial.
+- **Prial Instance** - we need a Piral instance, which connects to the feed service. Also, here we will utilize the Piral Instance, which we have created in the Getting Started tutorial.
 
-On the feed details page we get all sorts of information. Most notably, we see the currently available pilets. However, before we can publish a pilet, we need to obtain an API key. For this we can go to the API keys of the current feed (either from the homepage listing all feeds, or on the details page by clicking the keys icon on the feed bar).
+## Creating a Feed in the Piral Feed Service
```

Comment:
Can we please always have the blank lines before and after a heading?

Reply:
Corrected for all headings.